.aws/task_definition.json

```diff
@@ -29,28 +28,29 @@
             "environment": [],
             "environmentFiles": [
                 {
-                    "value": "arn:aws:s3:::quivr-env-variables/production.env",
+                    "value": "arn:aws:s3:::sr-vt-uw1-prod-env-bucket/production.env",
```

Comment:
use `arn:aws:s3:::sr-vt-env-variables/production.env` as we discussed. Thanks.

Reply:
also cleanup all other buckets please. 
we have ~ 10 buckets now, while we only need like 2-3.

Reply:
Even though we are deploying the different stacks in the same vpc/region, we still have to isolate the different set of resources, so we can mange the state with terraform or other IaC tools. So they will remain self contained and can be destroyed.

This is a start for later to deploy to different accounts or regions.

Let us share nothing between envs.

Reply:
Sure. Sounds like a plan. 👍

Reply:
Even though we now share a same region for the preview and prod env, but we should keep the resources sets for these envs completely isolated for future update. You could easy remove any resource in a env without affect others. And the deployment IaC code could be designed to treat any architecture as the same, no mater if we share a same region, or share a same account.